src/utils.py

```diff
@@ -104,7 +105,7 @@ def do_CV(X,y, model, multi_class=True, test_size=0.3):
     print(classification_report(y_test, y_pred))
     plot_confusion_matrix(y_test, y_pred)
     if multi_class == False:
-        my_dict = {'I':1, 'R':-1}
+        my_dict = {binary_class_labels[0]:1, binary_class_labels[1]:-1}
```

Comment:
I think I'd rather just keep this consistent instead of supporting both I/R and I/SR (cause it'll be easier down the line). You can just change this line to use I/SR and change line 776 with the replace S with R to replace both S and R with SR and I think it'll just work with both our things.